src/main.js

```diff
@@ -114,14 +116,12 @@ var quotes = [
   "A champion is defined not by their wins but by how they can recover when they fall."
 ];
 var savedPosters = [];
-var currentPoster;
+//var currentPoster;
```

Comment:
It looks like you have commented out this global variable of currentPoster. Is this still needed?